setup.py

```diff
@@ -892,6 +894,7 @@ def __init__(self, attrs=None):
         self.salt_spm_formula_dir = None
         self.salt_spm_pillar_dir = None
         self.salt_spm_reactor_dir = None
+        self.salt_home_dir = None
```

Comment:
You also need a `salt-hime-dir` entry in `SaltDistribution.global_options` or this value is not configurable at install time.

Reply:
`salt-home-dir`